testing/pom.xml

```diff
@@ -42,19 +37,11 @@
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
-      <version>${slf4j.version}</version>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>ch.qos.logback</groupId>
-      <artifactId>logback-core</artifactId>
-      <version>${logback.version}</version>
```

Comment:
In this module there is no OSGi, so we definitely don't need to declare logback-core, it's a transitive dependency of logback-classic.